wrapper/hermes_wrapper.cpp

```diff
@@ -0,0 +1,149 @@
+/* * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * *
+ * Distributed under BSD 3-Clause license.                                   *
+ * Copyright by The HDF Group.                                               *
+ * Copyright by the Illinois Institute of Technology.                        *
+ * All rights reserved.                                                      *
+ *                                                                           *
+ * This file is part of Hermes. The full Hermes copyright notice, including  *
+ * terms governing use, modification, and redistribution, is contained in    *
+ * the COPYING file, which can be found at the top directory. If you do not  *
+ * have access to the file, you may request a copy from help@hdfgroup.org.   *
+ * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * */
+
+#include "hermes_wrapper.h"
+
+#include "hermes.h"
+#include "bucket.h"
+#include "vbucket.h"
+
+extern "C" {
+
+std::shared_ptr<hermes::api::Hermes> hermes_ptr;
+
+int HermesInitHermes(char *hermes_config) {
+  int result = 0;
+
+  LOG(INFO) << "Hermes Wrapper: Initializing Hermes\n";
+
+  hermes_ptr = hermes::InitHermesDaemon(hermes_config);
+
+  return result;
+}
+
+void HermesFinalize() {
+  hermes_ptr->Finalize(true);
+}
+
+VBucketClass *HermesVBucketCreate(const char *name) {
+  LOG(INFO) << "Hermes Wrapper: Creating VBucket " << name << '\n';
+  hermes::api::Context ctx;
+  try {
+    hermes::api::VBucket *new_vbucket =
+      new hermes::api::VBucket(std::string(name), hermes_ptr, ctx);
```

Comment:
Memory leak. Also the `VBucket` never gets `Release`d or `Destroy`ed.

Reply:
I removed this VBucket wrapper since it is not used now.